### PR TITLE
New version: ProtonTechnologies.ProtonVPN version 3.0.5

### PR DESCRIPTION
**Pull request by superusercode**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107150)

### CODE REVIEW

**Review by russellbanks — DISMISSED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230517221936

File: manifests/p/ProtonTechnologies/ProtonVPN/3.0.5/ProtonTechnologies.ProtonVPN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /quiet
```
Comment (stephengillie) [inline manifests/p/ProtonTechnologies/ProtonVPN/3.0.5/ProtonTechnologies.ProtonVPN.installer.yaml:15]:
```suggestion
  Silent: /verysilent
  SilentWithProgress: /silent
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2023-05-17 20:17:37.650   InitializeSetup raised an exception (fatal). 
> 2023-05-17 20:17:37.650   Exception message:                           Runtime error (at 38:344):                           Internal error: ExtractTemporaryFile: The file "netcorecheck.exe" was not found. 
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230518034119

File: manifests/p/ProtonTechnologies/ProtonVPN/3.0.5/ProtonTechnologies.ProtonVPN.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ProtonTechnologies.ProtonVPN
+PackageVersion: 3.0.5
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
```
Comment (stephengillie) [inline manifests/p/ProtonTechnologies/ProtonVPN/3.0.5/ProtonTechnologies.ProtonVPN.installer.yaml:11]:
```suggestion
- interactive
- silent
```

**Comment by stephengillie** [comment]
> When run locally it works fine, though the progress is visible. It can error if you have .net core installed abnormally like i do (via scoop).

Thank you for the hint. Your WinGet client likely is running as `silentWithProgress`, while I believe mine is `silent`.

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > 2023-05-17 20:17:37.650   InitializeSetup raised an exception (fatal).
> > 2023-05-17 20:17:37.650   Exception message:                           Runtime error (at 38:344):                           Internal error: ExtractTemporaryFile: The file "netcorecheck.exe" was not found.

This error was unchanged. I'm not seeing it in manual validation, so I'm not sure of the best path forward here.

**Pull request closed without merging**